core/templates/App.ts

```diff
@@ -274,6 +274,26 @@ angular.module('oppia').factory('$exceptionHandler', [
       if (UNHANDLED_REJECTION_STATUS_CODE_REGEX.test(exception)) {
         return;
       }
+      // If an unhandled rejection is encountered in the AngularJS promise
+      // manager, it checks the data type of the rejection value. If it
```

Comment:
What is the "AngularJS promise manager"? Maybe elaborate more / provide a link / etc.

Also, here and elsewhere, you use "it" several times. I can't actually tell easily what each of the "it"s refers to. Perhaps dereference the pronoun and state the thing explicitly, for clarity?

Reply:
I've modified the explanation a bit and added relevant links per your comment.